databricks_cli/configure/cli.py

```diff
@@ -44,6 +47,16 @@ def _configure_cli_token(profile, insecure):
     update_and_persist_config(profile, new_config)
 
 
+def _configure_cli_aad_token(profile, insecure):
+    config = ProfileConfigProvider(profile).get_config() or DatabricksConfig.empty()
+    host = click.prompt(PROMPT_HOST, default=config.host, type=_DbfsHost())
+    token = os.environ.get('DATABRICKS_TOKEN')
+    az_token = os.environ.get('DATABRICKS_AZ_TOKEN')
```

Comment:
Do we have any guard against missing/invalid tokens? Seems like if users do are not guided to use the envvar approach yet, and if they don't write the variable correctly, the CLI would just silently fail by passing over the AZ token login option.

Also, it would be worth it to actually check-in with the PMs that env-var based approach would be acceptable.

Reply:
The CLI itself doesn't have any test against invalid tokens - it never did. Talked with PMs about env variable approach and they said it was acceptable

Reply:
e.g. this means that AAD tokens are not going to be refreshed by our cli and we may get errors for commands running for too long? MSAL/ADAL libs provide token refresh functionality.